packages/rrweb/test/integration.test.ts

```diff
@@ -590,6 +590,20 @@ describe('record integration tests', function (this: ISuite) {
     assertSnapshot(snapshots);
   });
 
+  it.only('should mask only inputs', async () => {
```

Comment:
is the `only` intentional?

Reply:
Oops, what happens when I don't have precommit linters